From bb4b66412a84cb4ad69436da382690883d5172e9 Mon Sep 17 00:00:00 2001
From: Martin Perina <mperina@redhat.com>
Date: Mon, 14 Apr 2014 08:26:59 +0200
Subject: [RHEL7 PATCH 1/7] Rename FENCE_KDUMP_CONFIG to FENCE_KDUMP_CONFIG_FILE

Renames FENCE_KDUMP_CONFIG variable to FENCE_KDUMP_CONFIG_FILE to
distinguish it from values read from fence_kdump_args option in
kdump.conf (introduced in following patches).

Related: rhbz#1086988
Signed-off-by: Martin Perina <mperina@redhat.com>
---
 dracut-kdump.sh        | 4 ++--
 dracut-module-setup.sh | 2 +-
 kdump-lib.sh           | 2 +-
 kdumpctl               | 4 ++--
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/dracut-kdump.sh b/dracut-kdump.sh
index d9e65ac..06e41bc 100755
--- a/dracut-kdump.sh
+++ b/dracut-kdump.sh
@@ -292,8 +292,8 @@ fence_kdump_notify()
     local nodes
 
     if [ -f $FENCE_KDUMP_NODES ]; then
-        if [ -f $FENCE_KDUMP_CONFIG ]; then
-            . $FENCE_KDUMP_CONFIG
+        if [ -f $FENCE_KDUMP_CONFIG_FILE ]; then
+            . $FENCE_KDUMP_CONFIG_FILE
         fi
 
         read nodes < $FENCE_KDUMP_NODES
diff --git a/dracut-module-setup.sh b/dracut-module-setup.sh
index bdadf7c..cd60cf1 100755
--- a/dracut-module-setup.sh
+++ b/dracut-module-setup.sh
@@ -444,7 +444,7 @@ kdump_check_fence_kdump () {
 
     echo "$nodes" > ${initdir}/$FENCE_KDUMP_NODES
     dracut_install $FENCE_KDUMP_SEND
-    dracut_install -o $FENCE_KDUMP_CONFIG
+    dracut_install -o $FENCE_KDUMP_CONFIG_FILE
 }
 
 # Install a random seed used to feed /dev/urandom
diff --git a/kdump-lib.sh b/kdump-lib.sh
index de32650..d72e16b 100755
--- a/kdump-lib.sh
+++ b/kdump-lib.sh
@@ -3,7 +3,7 @@
 # Kdump common variables and functions
 #
 
-FENCE_KDUMP_CONFIG="/etc/sysconfig/fence_kdump"
+FENCE_KDUMP_CONFIG_FILE="/etc/sysconfig/fence_kdump"
 FENCE_KDUMP_SEND="/usr/libexec/fence_kdump_send"
 FENCE_KDUMP_NODES="/etc/fence_kdump_nodes"
 
diff --git a/kdumpctl b/kdumpctl
index 5721f7f..398858d 100755
--- a/kdumpctl
+++ b/kdumpctl
@@ -249,8 +249,8 @@ function check_rebuild()
 	EXTRA_BINS="$EXTRA_BINS $CHECK_FILES"
 	files="$KDUMP_CONFIG_FILE $kdump_kernel $EXTRA_BINS"
 
-	if [ -f $FENCE_KDUMP_CONFIG ]; then
-		files="$files $FENCE_KDUMP_CONFIG"
+	if [ -f $FENCE_KDUMP_CONFIG_FILE ]; then
+		files="$files $FENCE_KDUMP_CONFIG_FILE"
 	fi
 
 	check_exist "$files" && check_executable "$EXTRA_BINS"
-- 
1.8.3.1

